storeshare_api/Gemfile

```diff
@@ -31,18 +33,18 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 gem 'bootsnap', require: false
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
-# gem "image_processing", "~> 1.2"
+gem 'image_processing', '~> 1.2'
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-gem "rack-cors", require: 'rack/cors'
+gem 'rack-cors', require: 'rack/cors'
 
 gem 'faker'
 
 gem 'pry-rails'
 
 gem 'devise-jwt'
 
-gem 'dotenv-rails', groups: [:development, :test]
+gem 'dotenv-rails', groups: %i[development test]
```

Comment:
just curious--what does this change do?

Reply:
just a fancy Ruby notation for a non-interpolated array of symbols, I think my style checker does all that for me lol